reference/FetchImportedModuleFromScriptCallBack.md

```diff
@@ -0,0 +1,22 @@
+User implemented callback to fetch imported modules dynamically in scripts.
+### Syntax 
+```
+typedef JsErrorCode (CHAKRA_CALLBACK * FetchImportedModuleFromScriptCallBack)(
+    _In_ JsSourceContext dwReferencingSourceContext,
+    _In_ JsValueRef specifier,
+    _Outptr_result_maybenull_ JsModuleRecord* dependentModuleRecord);
+```
+### Parameters 
+* __dwReferencingSourceContext__: The referencing script context that calls import()
+* __specifier__:  The specifier provided to the import() call.
+* __dependentModuleRecord__:  The ModuleRecord of the dependent module. If the module was requested before from other source, return the existing ModuleRecord, otherwise return a newly created ModuleRecord.
+
+### Return Value 
+Returns a **JsNoError** if the operation succeeded an error code otherwise.
```

Comment:
I suggest rephrasing this to `Returns **JsNoError** if the operation succeeded or an error code otherwise`